modules/k8s/kubectl_options.go

```diff
@@ -15,6 +15,8 @@ type KubectlOptions struct {
 	InClusterAuth bool
 	RestConfig    *rest.Config
 	Logger        *logger.Logger
+	// ConfigLogger is useful to set a different logger when configuring kubectl with a config. This can significantly reduce verbosity of tests
+	ConfigLogger *logger.Logger
```

Comment:
Hm, what will happen if the logger will not be provided?

Reply:
The logger falls back to the Default logger if it's nil. This also enables backward compatibility

https://github.com/gruntwork-io/terratest/blob/6fb86056797e3e62ebdd9011ba26605e0976a6f8/modules/logger/logger.go#L63-L70